crmsh/bootstrap.py

```diff
@@ -2016,10 +2047,8 @@ def update_nodeid(nodeid, node=None):
         csync2_update(corosync.conf())
         invoke("ssh -o StrictHostKeyChecking=no root@{} corosync-cfgtool -R".format(seed_host))
 
-    # if no SBD devices are configured,
-    # check the existing cluster if the sbd service is enabled
-    if not configured_sbd_device() and invoke("ssh -o StrictHostKeyChecking=no root@{} systemctl is-enabled sbd.service".format(seed_host)):
-        _context.diskless_sbd = True
+    if invoke("ssh -o StrictHostKeyChecking=no root@{} systemctl is-enabled sbd.service".format(seed_host)):
```

Comment:
Why don't you put this piece of code in the `SBDManager`? It totally belongs to sbd

Reply:
I put this into `detect_remote_sbd_service_status` method